### ISSUE TITLE
layouts just stops randomly

### ISSUE THREAD

**Issue by Arrow-x** [johnae/persway#62]
I am on Arch linux with `sway` 1:1.8.1-1 and `persway` 0.6.2
it works for a time and then stops it throughs this error
`2023-05-22T18:07:26Z ERROR persway::server::event_handlers::layout::stack_main] stack_main layout err: recursion limit exceeded at line 1 column 29391`
over and over

**Comment by johnae** [comment]
Interesting. Not using the scratchpad much myself. I'll see if I can reproduce this.

**Comment by rimeno** [comment]
Same behavior here, thanks Arrow-x to point the link with scratchpad.
Same setup : Arch, sway, persway 0.6.2.

**Comment by johnae** [comment]
@Arrow-x @rimeno Please try latest on main branch.

**Comment by Arrow-x** [comment]
nope, It is still an issue, I think I can reproduce it like this:
1 star the daemon `persway daemon -d stack_main`
2 in an empty workspace start a terminal and send it to the scratchpad 
3 start another terminal and send it to the scratchpad
4 start another terminal 
5 summon a scratchpad and resize it a bit (with the mouse) then put it back
6 bring the other one and resize it (there is a flicker in the size sometimes) and send it back
7 try starting another window will not stack now.

**Comment by rimeno** [comment]
Thanks Johnae, for simple scratchpad action it's now ok, but Arrow-x is also right, if scratchpad window is resised, the bug is still here.

**Comment by johnae** [comment]
Sorry for the delay. I'm just really busy. I'll get around to these at some point.
